subaccounts.go

```diff
@@ -4,34 +4,39 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+
+	"github.com/pkg/errors"
 )
 
 // https://www.sparkpost.com/api#/reference/subaccounts
 var SubaccountsPathFormat = "/api/v%d/subaccounts"
-var availableGrants = []string{
```

Comment:
Export these so 1) they're usable in tests and 2) they're end-user serviceable if needed